pdns/dnsdistdist/m4/dnsdist_enable_dnscrypt.m4

```diff
@@ -9,7 +9,7 @@ AC_DEFUN([DNSDIST_ENABLE_DNSCRYPT], [
   AM_CONDITIONAL([DNSCRYPT], [test "x$enable_dnscrypt" != "xno"])
 
   AM_COND_IF([DNSCRYPT], [
-    AM_COND_IF([LIBSODIUM], [
+    AS_IF([test x"$LIBSODIUM_LIBS" != "x"], [
```

Comment:
I'm not sure we actually need this change since it looks like the `LIBSODIUM` conditional is still defined?